testcases/contenttype.mqtt

```diff
@@ -0,0 +1,12 @@
+------------------ PUBLISH ---------------
+Topic Name: mytopic
+Content Type: application/xml
+--------------- User Properties ----------
+specversion: 0.4-wip
+id: 123
+source: https://cloudevents.io/contenttype
+type: io.cloudevents.contenttype-test
+datacontentencoding: Base64
```

Comment:
@clemensv is there an MQTT equivalent property for datacontentencoding?

Reply:
Searching for "encoding" in https://docs.oasis-open.org/mqtt/mqtt/v5.0/os/mqtt-v5.0-os.html produces 25 hits, all of which cover UTF-8 or property encoding, rather than payload encoding.

Reply:
@duglin No, there is no equivalent in MQTT. For MQTT 3.1.x, you even need to know the datacontenttype apriori as a convention on the topic. This particular field missing is not an issue, though, because the body is always a byte sequence.